apps/designer/app/routes/rest/props.$projectId.ts

```diff
@@ -9,8 +9,18 @@ export const loader: LoaderFunction = async ({
   params,
 }): Promise<Array<InstanceProps> | ErrorData> => {
   try {
-    const project = await db.project.loadById(params.projectId);
-    return await db.props.loadByProject(project, "production");
+    if (params.projectId === undefined) {
+      throw new Error(`Project ID required`);
+    }
+
+    const prodBuild = await db.build.loadProdByProjectId(params.projectId);
```

Comment:
wouldn't it read more nicely if we had build.loadByProjectId(projectId, 'prod') ?